internal/services/logic/logic_app_standard_resource.go

```diff
@@ -561,6 +581,7 @@ func resourceLogicAppStandardRead(d *pluginsdk.ResourceData, meta interface{}) e
 		d.Set("client_affinity_enabled", props.ClientAffinityEnabled)
 		d.Set("custom_domain_verification_id", props.CustomDomainVerificationID)
 		d.Set("virtual_network_subnet_id", props.VirtualNetworkSubnetID)
+		d.Set("public_network_access_enabled", props.PublicNetworkAccess)
```

Comment:
you will need to do a comparison here
```suggestion
		d.Set("public_network_access_enabled", props.PublicNetworkAccess == helpers.PublicNetworkAccessEnabled )
```

Reply:
thanks, I've accepted your suggestion.